mlflow/artifacts/__init__.py

```diff
@@ -52,7 +55,12 @@ def download_artifacts(
         return _download_artifact_from_uri(artifact_uri, output_path=dst_path)
 
     artifact_path = artifact_path if artifact_path is not None else ""
-    store = _get_store()
+
+    if tracking_uri is not None:
+        with _use_tracking_uri(tracking_uri):
+            store = _get_store()
+    else:
+        store = _get_store()
```

Comment:
```suggestion
    store = _get_store(tracking_uri)
```

Can we pass `tracking_uri` to `_get_store`?